lib/basedriver/driver.js

```diff
@@ -323,12 +323,22 @@ class BaseDriver extends Protocol {
       throw new errors.NotYetImplementedError();
     }
 
+    let unexpectedShutdownListener;
     const commandExecutor = async () => imgElId
       ? await ImageElement.execute(this, cmd, imgElId, ...args)
       : await B.race([
         this[cmd](...args),
-        new B((resolve, reject) => this.eventEmitter.on(ON_UNEXPECTED_SHUTDOWN_EVENT, reject))
-      ]);
+        new B((resolve, reject) => {
+          unexpectedShutdownListener = reject;
+          this.eventEmitter.on(ON_UNEXPECTED_SHUTDOWN_EVENT, unexpectedShutdownListener);
+        })
+      ]).finally(() => {
+        // https://github.com/appium/appium/issues/13631
```

Comment:
in general, IMO a comment which is only a link is not as useful as one which briefly explains something in words and then also includes a link.